server/index.js

```diff
@@ -9,3 +8,28 @@ app.listen(port, () => {
   console.log(`Karin's app listening at http://localhost:${port}`)
 });
 /*----------------------------------------------*/
+
+app.get('/reviews', (req, res) => {
+  database.fetchReviews((err, results) => {
+    if (err) {
+      res.status(404).send(err);
+    } else {
+      res.status(200).send(results);
+    }
+  });
+})
+
+// get review by product id
+app.get('/review', (req, res) => {
```

Comment:
Suggestion:
Is there a way for you to add product_id to the endpoint? For example, when the user looks reviews for product_id = 23. The endpoint would be /review/23.